ob1k-cache/src/main/java/com/outbrain/ob1k/cache/memcache/folsom/MemcachedClient.java

```diff
@@ -160,6 +174,7 @@ private boolean isOK(final MemcacheStatus memcacheStatus) {
             consumer.consume(Try.fromError(e));
           } catch (final ExecutionException e) {
             final Throwable error = e.getCause() != null ? e.getCause() : e;
+            log.error("Failed to run operation for " + cacheName + ", error: " + error.getMessage(), error);
```

Comment:
You should drop the redundant  `", error: " + error.getMessage()` - the exception already prints that
Also please use place holders for the name